admin/database/postgres/migrations/0044.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE billing_errors (
+    id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+    org_id UUID NOT NULL,
+    type INTEGER NOT NULL,
+    metadata JSONB NOT NULL DEFAULT '{}'::JSONB,
+    event_time TIMESTAMPTZ NOT NULL DEFAULT now(),
+    created_on TIMESTAMPTZ NOT NULL DEFAULT now(),
+    FOREIGN KEY (org_id) REFERENCES orgs (id) ON DELETE CASCADE,
+    CONSTRAINT billing_errors_org_id_type_unique UNIQUE (org_id, type)
+);
+
+CREATE TABLE billing_warnings (
```

Comment:
Since the schema is the same, would it make sense to merge this with `billing_errors` and using a column to determine if it's a warning – for example by incorporating the warning types in the `type` enum or adding something like a `level` or `warning BOOL` field instead? Presumably that would also make consuming the errors/warnings simpler downstream?